### DISCUSSION TITLE
Ошибки при обновлении

### DISCUSSION THREAD

**Discussion by Bullet57** [Anonym-tsk/nfqws-keenetic#96] [Обсуждения]
Кто знает, это что то критичное ? Вроде все работает, но при обновлении пишет ошибку.

Collected errors:
 * resolve_conffiles: Existing conffile /opt/etc/nfqws/user.list is different from the conffile in the new package. The new conffile will be placed at /opt/etc/nfqws/user.list-opkg.
 * resolve_conffiles: Existing conffile /opt/etc/nfqws/auto.list is different from the conffile in the new package. The new conffile will be placed at /opt/etc/nfqws/auto.list-opkg.
 * resolve_conffiles: Existing conffile /opt/etc/nfqws/exclude.list is different from the conffile in the new package. The new conffile will be placed at /opt/etc/nfqws/exclude.list-opkg.

**Comment by Xassewi** [comment]
Все норм. стоявшие списки доменов заменены новыми.

**Reply by pwgms213** [reply]
А не наоборот? новые сохранены как .list-opkg

**Reply by LordSlon** [reply]
> Все норм. стоявшие списки доменов заменены новыми.

The new conffile will be placed at /opt/etc/nfqws/exclude.list-opkg. - разве это не значит, что старый файл остался, а новый с обновлением записан в opkg отдельно?

**Reply by pwgms213** [reply]
Именно так, но я проверил оба файла настроек, в моем случае не было никаких различий, возможно у кого-то будут, учитывая ченжлог.

**Comment by LordSlon** [comment]
Нигде не указано что делать с этими файлами. Если кто объяснит этот момент подробнее, буду благодарен

**Reply by pwgms213** [reply]
Если работает , можно ничего не делать, а вообще надо старые удалить а .list-opkg переименовать в .list

**Reply by LordSlon** [reply]
> Если работает , можно ничего не делать, а вообще надо старые удалить а .list-opkg переименовать в .list

а если например переустановить пакет, они перезапишутся? Или хвосты останутся?

**Reply by pwgms213** [reply]
чтоб не мучиться, поставьте файловый менеджер mc (opkg install mc).
Затем впишите mc (запустится файловый менеджер похожий на FAR).
Будет проще работать с файлами.
Для работы с листами и конфигами советую поставить nano-plus (opkg install nano-plus).  Для открытия любого конфига или листа
nano-plus /opt/etc/nfqws/nfqws.conf (пример редактирования конфига).
У себя отключил режим авто, оcтавил только user и в него добавил все домены из zapret-discord-youtube. Работало и с авто, но так мне кажется удобнее.
